example_config_class.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class ThingWorker
+  def self.process(message)
+    FastlyNsq.logger.info("IN PROCESS: #{message}")
+  end
+end
+
+FastlyNsq.configure do |config|
+  config.channel = 'william'
+  # config.logger = Logger.new
+  # config.preprocessor = ->(message) { FastlyNsq.logger.info 'PREPROCESSESES' }
+
+  config.listen_to do |topics|
+    topics.add('assign_supplemental_plans', ThingWorker)
+  end
```

Comment:
I was going to say that there is the chance you could want to listen to the same topic several times
```ruby
[
  { topic: 'thing', klass: Doer },
  { topic: 'thing', klass: OtherDoer },
]
```
I feel like this will be something we eventually want this to be able to accomplish... However, to have that actually work several messages, you would need different channels. That would mean more work here as you would have to push the channel configuration down to the Listener rather than being global. Additionally, you can accommodate that now by just running two daemons...